bip-segwit-uasf.mediawiki

```diff
@@ -0,0 +1,59 @@
+<pre>
+  BIP: ?
+  Layer: Consensus (soft fork)
+  Title: Segwit deployment with versionbits and guaranteed lock-in
+  Author: Shaolin Fry <shaolinfry@protonmail.ch>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-????
+  Status: Draft
+  Type: Standards Track
+  Created: 2017-04-14
+  License: BSD-3-Clause
+           CC0-1.0
+</pre>
+
+==Abstract==
+
+This document specifies a user activated soft fork for [[bip-0141.mediawiki|BIP141]], [[bip-0143.mediawiki|BIP143]] and [[bip-0147.mediawiki|BIP147]] using versionbits with guaranteed lock-in.
+
+==Motivation==
+
+Miners have been reluctant to signal the BIP9 segwit deployment despite a large portion of the Bitcoin ecosystem who want the soft fork activated. This BIP specifies a user activated soft fork (UASF) that deploys segwit again using versionbits with guaranteed lock-in on timeout if the BIP is not already locked-in or activated by the timeout. This ensures users have sufficient time to prepare and no longer require a miner supermajority, while still allowing for an earlier miner activated soft fork (MASF).
+
+==Reference implementation==
+
+https://github.com/bitcoin/bitcoin/compare/master...shaolinfry:uasegwit-flagday
+
+==Specification==
+
+This deployment will set service bit (1<<5) as NODE_UAWITNESS.
+
+==Deployment==
+
+This BIP will be deployed by BIP8 with the name "uasegwit" and using bit 2.
```

Comment:
It's unclear to be if each BIP9 deployment needs a completely different common name or not, but in any case, since we need a new service bit, I think there has to be a new name anyway. I'm going to leave this as is for the time being.